MarkdownKit/Sources/Common/Elements/Link/MarkdownLink.swift

```diff
@@ -11,8 +11,11 @@ open class MarkdownLink: MarkdownLinkElement {
 
   fileprivate static let regex = "(\\[[^\\]]+)(\\]\\([^\\s]+)?\\)"
 
+  private let schemeRegex = "([a-z]{2,20}):\\/\\/"
```

Comment:
I like that, we could now use a specific app-related scheme in some link.

Reply:
Yep, or tel:// and mailto://